next.config.ts

```diff
@@ -1,7 +1,7 @@
 import type { NextConfig } from "next";
 
 const nextConfig: NextConfig = {
-  /* config options here */
+  reactStrictMode: true,
```

Comment:
Seems unrelated to what we're doing but I think it's still a good call adding this